src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's address in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ */
+public class Block {
+
+    public static final String EXAMPLE = "123";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Block can be in any format";
```

Comment:
Is this an Address constraint? Change variable name to a more appropriate one. 
Same comment for validation regex